web/src/main/react-dashboard/src/components/TestRunView.js

```diff
@@ -49,563 +42,570 @@ import {FAIL, SUCCESS, ERROR, PENDING, RUNNING, HTTP_UNAUTHORIZED, LOGIN_URI, TE
-                    headers: {
-                      'Accept': 'application/json'
-                    }
-                  })
-                  .then(this.handleError)
-                  .then(response => {
-                    this.setState({
-                      showLogDownloadErrorDialog: !response.ok
-                    });
-                    return response.json();
-                  }).then(data => {
-                    if (!this.state.showLogDownloadErrorDialog) {
-                      Download(data.inputStreamContent, "test-run.log", "plain/text");
+        return (
+            <div>
+                {/*Sub header*/}
+                {this.state && this.state.currentInfra && (() => {
+                    const subHeader = (<td style={{padding: 5}}>
+                        <i> {this.state.currentInfra.relatedProduct}/
+                            {this.state.currentInfra.relatedDeplymentPattern} /
+                            {this.state.currentInfra.infraParameters}</i>
+                    </td>);
+                    switch (this.state.currentInfra.testPlanStatus) {
+                        case FAIL:
+                            return <Subheader style={{
+                                fontSize: '20px',
+                                backgroundColor: "#ffd6d3"
+                            }}>
+                                <Table responsive>
+                                    <tbody>
+                                    <tr>
+                                        <td style={{padding: 5}}>
+                                            <img
+                                                src={require('../close.png')}
+                                                style={{
+                                                    verticalAlign: "middle",
+                                                    height: "50px",
+                                                    width: "50px"
+                                                }}/>
+                                        </td>
+                                        {subHeader}
+                                    </tr>
+                                    </tbody>
+                                </Table>
+                            </Subheader>;
+                        case SUCCESS:
+                            return <Subheader style={{
+                                fontSize: '20px',
+                                backgroundColor: "#cdffba"
+                            }}>
+                                <Table responsive>
+                                    <tbody>
+                                    <tr>
+                                        <td style={{padding: 5}}>
+                                            <img
+                                                src={require('../success.png')}
+                                                style={{
+                                                    verticalAlign: "middle",
```

Comment:
Shall we specify classes for these elements and write the styles in a separate css file? That way we will not need to duplicate the styles. We can do it as a separate effort btw.

Reply:
+1 yes, we have to move this type of repeating styling into a separate CSS file. This fix resolves formatting issues and +1 for addressing this in a separate effort. Further, IMO we can use icons instead of these images. WDYT

Reply:
Let's raise a ticket and handle it elsewhere. We will have new set of statuses pretty soon as well.